tools/file-testing/file_test.py

```diff
@@ -9,7 +9,7 @@
 
 # Requirements:
 #  Python 2.6 or later
-#  OSSEC 2.8 or later
+#  Wazuh 2.8 or later
```

Comment:
OSSEC V2.8 was released in 2014 and Wazuh started in 2015. The first documented version of Wazuh (2.1) was released in 2017.
I think we can eliminate this requirement.

Reply:
You are right. Removed!